### PR TITLE
Merge changes from modulerelease520 to master

### LINKED ISSUE #464: Merge in changes from release 5.2.0 branch to master
## Story
The modulerelease520 branch was created in December and includes some changes/fixes that we want to move into the master branch. Since the 5.2.0 module release is delayed, I will delete the modulerelease520 branch after this is merged and re-create it when the release is ready.

### PR DESCRIPTION
**Pull request by gkowalski-google**
Resolves #464

### CODE REVIEW

File: modules/on_gke/README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ This sub-module deploys Forseti on GKE.  In short, this deploys a server contain
 | groups\_settings\_violations\_should\_notify | Notify for groups settings violations | bool | `"true"` | no |
 | groups\_violations\_should\_notify | Notify for Groups violations | bool | `"true"` | no |
 | gsuite\_admin\_email | G-Suite administrator email address to manage your Forseti installation | string | `""` | no |
-| helm\_chart\_version | The version of the Helm chart to use | string | `"2.1.0"` | no |
+| helm\_chart\_version | The version of the Helm chart to use | string | `"2.2.0"` | no |
```
Comment (gkowalski-google) [inline modules/on_gke/README.md]: Will revert this to 2.1.0 with a commit once the tests complete.

File: modules/on_gke/variables.tf [hunk 2]
```diff
@@ -882,7 +882,7 @@ variable "git_sync_wait" {
 
 variable "helm_chart_version" {
   description = "The version of the Helm chart to use"
-  default     = "2.1.0"
+  default     = "2.2.0"
```
Comment (gkowalski-google) [inline modules/on_gke/variables.tf]: Will revert this to 2.1.0 with a commit once the tests complete.

**Comment by gkowalski-google** [comment]
@paskal Can you please add a `@googlebot I consent.` if you are OK with your commit being included in this PR?

**Comment by paskal** [comment]
@googlebot I consent.

**Pull request merged**